plugins/coatils.py

```diff
@@ -93,7 +93,7 @@ def contrib_stats(self, msg, match):
                 len(Coatils.all_langs())
             )
         # stats
-        elif stat_type == '' and entity == '':
+        else:
```

Comment:
https://github.com/coala/corobo/pull/613/files#r208848778

Reply:
This isnt as simple as the other one.  Each branch yields, and then goes to the end.

To remove this else branch, we would need to add returns to all of the other branches, which isnt very helpful.